.travis.yml

```diff
@@ -1,5 +1,7 @@
 dist: trusty
 
+sudo: required
```

Comment:
will need to add docker as a service to here too:
```
sudo: required

services:
  - docker
```

Reply:
done, thanks!